content/apps/catalogs.md

```diff
@@ -112,155 +113,45 @@ Once you have an enabled catalog, paginate the related products using this [endp
+After completing this tutorial, you'll be able to push your JSON mapping schema to the PIM and get product data in the format you expect.
 
 ### Requirements
 
 - You have access to an Akeneo PIM ([get your App developer starter kit](/apps/overview.html#app-developer-starter-kit))
-- Your app already manages the authorization process 
+- Your app already manages the authorization process
 - You followed the [Getting started with catalogs](#getting-started-with-catalogs) steps
 
 ### Step 1: Define your JSON schema
 
-The first step to using the mapping feature is determining the JSON schema you need to push to the Akeneo PIM to get mapped product data. 
+The first step to using the mapping feature is determining the JSON schema you need to push to the Akeneo PIM to get mapped product data.
 
-::: info 
+::: info
 **JSON Schema is a declarative language that allows annotating and validating JSON documents.** It describes an existing data format, provides clear human- and machine-readable documentation, and allows to validate data which is useful for ensuring the quality of client-submitted data.
 :::
 
-To help you define your schema, we advise you to use this online validator pre-configured with our latest meta-schema: [jsonschemavalidator.net](https://www.jsonschemavalidator.net/s/D85OL1LE). The validator highlights errors if there are some or displays a success message if your schema matches all our meta-schema constraints. 
-
-You can also download the latest meta-schema at this url: [product mapping meta-schema - v0.0.13 (May, 2023)](/mapping/product/0.0.13/schema)
-
-JSON schema example: 
+To help you define your schema, we advise you to use this online validator pre-configured with our latest meta-schema: [jsonschemavalidator.net](https://www.jsonschemavalidator.net/s/sj7VwD3l). The validator highlights errors if there are some or displays a success message if your schema matches all our meta-schema constraints.
 
-```
-{
-  "$id": "https://example.com/product",
-  "$schema": "https://api.akeneo.com/mapping/product/0.0.12/schema",
-  "$comment": "My schema !",
-  "title": "Product Mapping",
-  "description": "JSON Schema describing the structure of products expected by our application",
-  "type": "object",
-  "properties": {
-    "uuid": {
-      "title": "Product UUID",
-      "type": "string"
-    },
-    "type": {
-      "title": "Product type",
-      "type": "string"
-    },
-    "sku": {
-      "title": "SKU (Stock Keeping Unit)",
-      "description": "Selling Partner SKU (stock keeping unit) identifier for the listing. \n SKU uniquely identifies a listing for a Selling Partner.",
-      "type": "string"
-    },
-    "name": {
-      "title": "Product name",
-      "type": "string"
-    },
-    "body_html": {
-      "title": "Description (textarea)",
-      "description": "Product description in raw HTML",
-      "type": "string",
-      "minLength": 0,
```

Comment:
Here we have on the old example some constraints, we are missing those in the new example file.